### ISSUE TITLE
Documentation outdate for 2CKAN 2.9

### ISSUE THREAD

**Issue by andrecastro0o** [ckan/ckanext-harvest#423]
Am I right to assume that the https://github.com/ckan/ckanext-harvest/tree/v1.3.1#configuration step paster instruction is outdated in CKAN 2.9?
 
`(pyenv) $ paster --plugin=ckanext-harvest harvester initdb --config=/etc/ckan/default/production.ini`

should be  `(pyenv) $ ckan --config=/etc/ckan/default/ckan.ini harvester initdb`

**Comment by amercader** [comment]
Yes, that's correct @andrecastro0o . Do you want to submit a PR including the correct command for the different CKAN versions?

**Comment by andrecastro0o** [comment]
@amercader that's a good idea :parrot: 
See https://github.com/ckan/ckanext-harvest/pull/424
